plugin/wakatime.vim

```diff
@@ -287,6 +287,16 @@ EOF
             if s:GetIniSetting('settings', 'api_key') != '' || s:GetIniSetting('settings', 'apikey') != ''
                 let found_api_key = s:true
             endif
+
+            if !found_api_key
+                let vault_cmd = s:GetIniSetting('settings', 'api_key_vault_cmd')
+                if vault_cmd != ''
```

Comment:
`if !empty(vault_cmd)`

Reply:
https://vimhelp.org/builtin.txt.html#empty%28%29

Reply:
Is there a guideline to use `empty` function? There are other lines of code that's not using it, like
```
if s:GetIniSetting('settings', 'api_key') != '' || s:GetIniSetting('settings', 'apikey') != ''
```

Reply:
That's just legacy code, we can change it to also use `empty` later.